data/tlg4089/tlg004/tlg4089.tlg004.1st1K-grc1.xml

```diff
@@ -43,16 +42,20 @@
             <sourceDesc>
                 <biblStruct>
                     <monogr>
-                        <title>Patrologiae Graecae Tomus LXXXII</title>
-                        <title>Theodoretus Cyrensis Episcopus Opera Omnia, Tomus Tertius</title>
+                        <title xml:lang="lat">Theodoretus Cyrensis Episcopus Opera Omnia</title>
                         <author>Theodoret, Bishop of Cyrus</author>
-                        <editor>Joan. Ludov. Schulze</editor>
+                        <editor>Johann Ludvig Schulze</editor>
```

Comment:
Middle name should be "Ludwig", I think. At least that is what it is in the `<editor>` element above.